docs/source/devguide.md

```diff
@@ -259,54 +259,116 @@ Make sure your change
-* Goto [xcube/releases](https://github.com/dcs4cop/xcube/releases) and press button "Draft a new Release".
+* Go to [xcube/releases](https://github.com/dcs4cop/xcube/releases) 
+  and press button "Draft a new Release".
   - Tag version is: `v${version}` (with a "v" prefix)
-  - Release title is: `${version}` 
+  - Release title is: `${version}` (without a "v" prefix) 
   - Paste latest changes from `CHANGES.md` into field "Describe this release"
   - Press "Publish release" button
 * After the release on GitHub, rebase sources, if the branch was `master`, 
   create a new maintenance branch (see above)
-* In `xcube/version.py` increase version number and append a `.dev0` suffix to version name so 
-  that it is still PEP-440 compatible.
+* In `xcube/version.py` increase version number and append a `.dev0` suffix 
+  to the version name so that it is still PEP-440 compatible.
+* Adjust version in `Dockerfile` accordingly.
 * In `CHANGES.md` add a new version headline and attach ` (in development)` to it.
 * Push changes to either master or a new maintenance branch (see above).
 * Activate new doc version on ReadTheDocs. 
 
-Go through the same procedure for all xcube plugin packages dependent on this version of xcube.
+Go through the same procedure for all xcube plugin packages 
+dependent on this version of xcube.
+
+### Release on Conda-Forge
+
+These instructions are based on the documentation at 
+[conda-forge](https://conda-forge.org/docs/maintainer/updating_pkgs.html).
+
+Conda-forge packages are produced from a github feedstock repository belonging 
+to the conda-forge organization. A repository's feedstock is usually located at 
+`https://github.com/conda-forge/<repo-name>-feedstock`, e.g., 
+https://github.com/conda-forge/xcube-feedstock.
+The package is updated by 
+* forking the repository
+* creating a new branch for the changes
+* creating a pull request to merge this branch into conda-forge's feedstock repository. 
+
+The first of these steps is usually already done. 
+You may find forks at `https://github.com/dcs4cop/<repo-name>-feedstock` . 
+
+In detail, the steps are:
+
+1. Update the dcs4cop fork of the feedstock repository, if it's not already 
+   up to date with conda-forge's upstream repository.
+
+1. Clone the repository locally and create a new branch. The name of the branch 
+   is not strictly prescribed, but it's sensible to choose an informative name like 
+   `update_0_5_3`.
+
+1. Rerender the feedstock using conda-smithy. This updates common conda-forge 
+   feedstock files. It's probably easiest to install conda-smithy in a 
+   fresh environment for this:
+
+   `conda install -c conda-forge conda-smithy`
+    
+   `conda smithy rerender -c auto`
+
+   It's also possible to have the rendering done by a bot as part of the pull request,
+   but this doesn't seem to work very reliably in practice.
+
+1. Update `recipe/meta.yaml` for the new version. 
```

Comment:
Also this will be done by the bot automatically if build eq 0.

Reply:
Not clear. Do you argue that this sentence "It's also possible to have the rendering done by a bot as part of the pull request,
   but this doesn't seem to work very reliably in practice." is wrong?

Reply:
I wrote that sentence based on my personal experience (see comments on https://github.com/conda-forge/xcube-feedstock/pull/9 ). It was some kind of permissions problem: the bot didn't have write access to commit its changes, and requested that I tick the "Allow edits from maintainers" box, but this box didn't even appear for me. I'm sure there was some way around it but I found it easier just to do the rerendering locally.